keras_cv/layers/object_detection3d/heatmap_decoder.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.object_detection3d import voxel_utils
+
+
+def decode_bin_heading(predictions: tf.Tensor, num_bin: int) -> tf.Tensor:
+    """Decode bin heading.
+
+    Computes the box heading (orientation) by decoding the bin predictions. The
+    predictions should contain bin classification scores (first num_bin scores)
+    and corresponding bin residuals (the following num_bin scores).
+
+    Args:
+      predictions: Prediction scores tensor with size [N, num_bin*2] predictions =
+        [:, bin_1, bin_2, ..., bin_k, res_1, res_2, ..., res_k], where k is the
+        number of bins and N is the number of boxes
```

Comment:
Missing period (.). Many docstrings in this PR miss period.

Reply:
Added